charts/deepgram-self-hosted/CHANGELOG.md

```diff
@@ -8,6 +8,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
 ### Added
 
+### Changed
+
+### Fixed
+
+## [0.6.0] - 2024-09-27
```

Comment:
Let's add the embedded link to this release at the bottom of the Changelog.

Reply:
Added [here](https://github.com/deepgram/self-hosted-resources/pull/44/commits/d3888047098ca16ae40ec6b032f7729f13d13916)!